bin/rebuild

```diff
@@ -16,6 +16,25 @@
 #     (mkdir versiondb; cd versiondb; git init; mkdir dep_db ver_db manifests)
 #
 
+# true if `lsst-build prepare` has run successfully
+PREPARED=false
+
+#
+# Add 128 to non-zero exit statuses if the `lsst-build prepare` subcommand has
+# completed.  This is to allow consumer scripts to be able to determine if a
+# failure in this script occurred before or after the prepare step.
+#
+cleanup() {
+    local rv=$?
+
+    [[ $rv == 0 ]] && exit
+    [[ $PREPARED == true ]] && rv=$((rv + 128))
```

Comment:
I thought about sanity checking it but didn't bother since 2 is the highest value in usage.  I'll fix it up.